Resources/Prototypes/Catalog/uplink_catalog.yml

```
@@ -129,6 +129,13 @@
   icon: /Textures/Objects/Tools/emag.rsi/icon.png
   price: 8
 
+- type: uplinkListing
+  id: UplinkHypopen
+  category: Utility
+  itemId: Hypopen
+  description: A chemical hypospray disguised as a pen, capable of instatly injecting up to 15u of reagents. Starts empty.
```

```suggestion
  description: A chemical hypospray disguised as a pen, capable of instantly injecting up to 15u of reagents. Starts empty.
```

PLEASE spellcheck your PRs